src/webaudio.js

```diff
@@ -12,9 +12,14 @@ WaveSurfer.WebAudio = {
 
     getAudioContext: function () {
         if (!this.ac) {
-            this.ac = new (
-                window.AudioContext || window.webkitAudioContext
-            );
+            // check if audiocontext limit has been reached
+            if (this.ac.length >= 5) {
```

Comment:
I don't think we should hardcode any specific number. It probably depends on the browser. Just catch the error the browser is throwing when instantiating AudioContext.